strict-containers/src/Data/Strict/ContainersUtils/Autogen/StrictPair.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE CPP #-}
+#if !defined(TESTING) && defined(__GLASGOW_HASKELL__)
+{-# LANGUAGE Safe #-}
+#endif
+
+#include "containers.h"
+
+-- | A strict pair
+
+module Data.Strict.ContainersUtils.Autogen.StrictPair (StrictPair(..), toPair) where
```

Comment:
unfortunate duplication of what is already in `strict`

Reply:
Yes, we could file a PR to them to use `strict`, then re-running `regen.sh` will automatically pull this in and drop it here too.